from_mwt_ds/DataScience/vw_executor/vw.py

```diff
@@ -125,10 +126,52 @@ def __init__(self,
         self.status = ExecutionStatus.NotStarted
         self.model_file = model_file
         self.model_folder = model_folder
+        self._order_position = order_position
         self._no_run = no_run
         self.args = self._prepare_args(self.job.cache)
         self.start_time = None
         self.end_time = None
+    
+    def create_human_readeable_symlink(self, translate_output: Dict[str, str] = {"-p": "predictions.txt", "-c": "cache", "-f": "final_regressor.vwmodel", "--extra_metrics": "extra_metrics.json"}) -> None:
+        import os
+        from datetime import datetime
+
+        if os.name == "nt":
+            def symlink_ms(source, link_name):
+                from subprocess import call
+                call(['mklink', link_name, source], shell=True)
+            os.symlink = symlink_ms
+        
+        def remove_argdash(arg: str):
+            if len(arg) > 1 and arg[0] == "-" and arg[1] == "-":
+                return arg[2:]
+            elif arg[0] == "-":
+                return arg[1:]
+            else:
+                return arg
+        
+        def clean_args(self):
+            args = self.args.split()
+            for i in range(len(args)-1, 0, -1):
+                if args[i] in self.outputs.keys() or args[i] in {"-i", "-d"}:
+                    args.pop(i+1)
+                    args.pop(i)
+
+            args = [remove_argdash(arg) for arg in args]
+            return ".".join(args)
+        
+        argdirname = clean_args(self)
```

Comment:
isn't job.name working here?

Reply:
actually that should work

Reply:
I mean, what is the benefit over:
argdirname = self.job.name
?